packages/sit-onyx/src/i18n/index.ts

```diff
@@ -16,6 +16,8 @@ type GetTypeOfTranslations<T> = T extends object
 /** Available translations that are used by onyx components. */
 export type OnyxTranslations = GetTypeOfTranslations<typeof enUS>;
 
+export type OnyxTranslationKeys = FlattenedKeysOf<OnyxTranslations>;
```

Comment:
```suggestion
export type OnyxTranslationKey = FlattenedKeysOf<OnyxTranslations>;
```

Its a union type, no array so I'd say its a single key type

Reply:
sounds good